### PR TITLE
improve(onboarding-ui): Request Trial Page Updates

### PR DESCRIPTION
**Pull request by ujorgeleite**
add new size for some texts
add new footer description with action on the page
add some new texts

- [x] I have read the Contributing Guide
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have labeled the PR correctly with the related package
- [x] I have run Loki's visual regression tests (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Issue(s)

[Task](https://app.clickup.com/t/1mzxxza)

### CODE REVIEW

File: packages/onboarding-ui/src/common/PlanFeatureIcon.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const DescriptionIcon = ({
 }): ReactElement => (
   <svg
     width='13'
-    height='9'
+    height='s1'
```
Comment (dougfabris) [inline packages/onboarding-ui/src/common/PlanFeatureIcon.tsx:10]: I don't get it

Comment (ujorgeleite) [reply packages/onboarding-ui/src/common/PlanFeatureIcon.tsx:10]: Is it to put a new pattern on the size of the SVG, we shouldn't  use this directly on svg?

Comment (dougfabris) [reply packages/onboarding-ui/src/common/PlanFeatureIcon.tsx:10]: I don't think that **s1** is a recognizable value for height in svg elements

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: packages/onboarding-ui/src/common/PlanFeatureIcon.tsx [hunk 2]
```diff
@@ -7,7 +7,6 @@ const DescriptionIcon = ({
 }): ReactElement => (
   <svg
     width='13'
-    height='9'
```
Comment (dougfabris) [inline packages/onboarding-ui/src/common/PlanFeatureIcon.tsx:10]: lets keep this `height` value

**Review by dougfabris — CHANGES_REQUESTED** [review]

**Review by dougfabris — APPROVED** [review]

File: packages/onboarding-ui/src/pages/RequestTrialPage/RequestTrialPage.tsx [hunk 3]
```diff
@@ -26,6 +25,9 @@ const RequestTrialPage = (props: RequestTrialPageProps): ReactElement => {
         subtitle={t('page.requestTrial.subtitle')}
       >
         <RequestTrialForm {...props} />
+        <ManageWorkspaceFallback
+          onManageWorkspaceClick={props.onManageWorkspaceClick}
+        />
```
Comment (geekgonecrazy) [inline packages/onboarding-ui/src/pages/RequestTrialPage/RequestTrialPage.tsx:30]: This was not supposed to come back.  This was intentionally removed: https://github.com/RocketChat/Rocket.Chat.Fuselage/pull/566/files

**Pull request merged**